gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/DiffDependencyTaskConfig.groovy

```diff
@@ -0,0 +1,11 @@
+package com.novoda.gradle.diffdependencytask
+
+import com.novoda.gradle.diffdependencytask.git.ChangedFilesProvider
+
+class DiffDependencyTaskConfig {
+
+    public GroovyCallable<List<String>> changedFilesProvider = ChangedFilesProvider.create()
+
+    public ConditionalDependencyRepository conditionalDependencyRepository = new MemoryConditionalDependencyRepository()
```

Comment:
You're right. The convention is to use the `*Extension` suffix in this case tho. That and reviewing gradle code too late was the cause of my comment. You can just ignore it ;)